masterlist.yaml

```diff
@@ -9221,6 +9221,20 @@ plugins:
       - crc: 0xFF60C24A
         util: 'SSEEdit v4.0.3'
 
+  - name: 'Faultier''s PBR landscapes.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/125308/' ]
+    after:
+      - 'Better Dynamic Snow SE.esp'
+      - 'BlendedRoads.esp'
+    dirty:
+      - <<: *quickClean
+        crc: 0x9F33E185
+        util: '[TES5VREdit v4.1.5f](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 1
+    clean:
```

Comment:
We usually don't add `clean` data, if the plugin is `dirty`. As such, please remove the `clean` data.

Reply:
I removed the clean data as requested.

Reply:
Sorry, I should have been more clear. It's not that the whole cleaning report needs to be removed, only the `clean` part of it. So you could do it like this:
```yaml
  - name: 'Faultier''s PBR landscapes.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/125308/' ]
    after:
      - 'Better Dynamic Snow SE.esp'
      - 'BlendedRoads.esp'
    dirty:
      - <<: *quickClean
        crc: 0x9F33E185
        util: '[TES5VREdit v4.1.5f](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 1
```